samples/nodejs-typescript-food-catalog/teamsapp.yml

```diff
@@ -87,3 +91,4 @@ publish:
       appPackagePath: ./appPackage/build/appPackage.${{TEAMSFX_ENV}}.zip
     writeToEnvironmentFile:
       publishedAppId: TEAMS_APP_PUBLISHED_APP_ID
+projectId: 0d487eb7-2637-4696-91cb-696cea91cd70
```

Comment:
Should this be committed?

Reply:
No, I'll remove it when the PR is ready 👍